app/frontend/src/elm/Component/Main/Page/Resource/Undelegate.elm

```diff
@@ -437,8 +438,9 @@ validateEach accountValidation cpuQuantityValidation netQuantityValidation =
 validate : Model -> String -> String -> Model
 validate ({ undelegatebw } as model) unstakePossibleCpu unstakePossibleNet =
     let
+        -- NOTE(boseok): because receiver must be one of delegate list accounts, requestStatus is passed as Succeed
```

Comment:
Good comment!
It would be better if you write the comment in a complete sentence starting with capital letter and ending with period.

e.g. Because ... Succeed.

Reply:
will do like that :)